docs/src/user_guide/usage/filters.md

```diff
@@ -0,0 +1,39 @@
+# Filters
+
+## am-highlight-morphs
+
+Highlights morphs based on your learning progress, just-in-time.
+
+If there are cards that you do not wish to add the
+[am-highlighted](../setup/settings/extra-fields.md#using-am-highlighted) field to, but you still would like to
+highlight the morphs, even from cards that you do not process with AnkiMorphs, you can use the `am-highlight-morphs`
+card template filter. It is intended to result in the same highlight capability as the
+`am-highlighted` extra field. It does however incur a small overhead for the just-in-time processing. It is not
+available on mobile.
+
+To use the filter, you will need to register [a note filter](note-filter.md#read--modify) for each note type
+that you would like to use the `am-highlight-morphs` filter on. If you do not want this card to be considered for
+your learning progress, you can disable read and modify flags. With this setup step done, you can start adding the
+filter to cards for that note type.
+
+as with
```

Comment:
Note to self I think this is errant and should be removed, look when you're back at your pc. 

```suggestion
```